core/src/main/resources/inetsoft/util/srinter.properties

```diff
@@ -262,6 +262,7 @@ Change\ from\ previous\ year\ of=Change from previous year of
 !
 AFM\ Font\ Path=AFM Font Path
 ANSI\ Join\ Syntax=ANSI Join Syntax
+API\ License\ Key=API License Key
```

Comment:
Change to "API Key", this is not a license except in terms of internal implementation and we should not use that term as it will be confusing to users.